polyform.md

```diff
@@ -50,6 +50,10 @@ It is a permitted purpose to evaluate whether the software suits a particular ap
 
 It is a permitted purpose to use the software for the internal business operations of you and your company.
 
+## NX: Noncompete
+
+It is not a permitted purpose to provide any offering that competes with the software or any other offering the licensor provides using the software.  However, if you are using the software to provide an offering that does not compete, and the licensor begins providing a competing offering using the software, you may continue using versions of the software available under these terms at that time to provide your offering, but not any later versions made available under these terms.
```

Comment:
What does compete mean - is that defensible?

Reply:
I don't think there's any enforcement problem with "compete", any more than "software" or other words used throughout the license.

Courts typically look first to whatever definition the terms provide, then to any legal definition, if it's a legal term, then to how the industry uses the term, if it has a peculiar meaning in industry, and then to everyday meaning, by citing dictionaries and such.

Competition shows up in commercial contracts and governance documents all the time. It can be unclear in specific circumstances, but that doesn't mean the courts throw the terms out. Instead, the parties involved cite facts about the situation to try to answer the question. Courts and juries weigh the evidence and decide.